### ISSUE TITLE
Incorrect notes when right clicking in Taiko Editor

### ISSUE THREAD

**Issue by ArztVielfrass** [ppy/osu#12870]
It's better to explain if you reproduce it yourself, no need to bother with screenshots.

1. Create a new beatmap
2. Insert a music resource
3. Set Timing
4. Activate Hit Tool
5. Right click on the main editor
> A blue kat note appears
6. Play the newly added note
> It should play the kat with the kat sound
7. Save your masterpiece
8. Play your masterpiece ( Not in the Editor )
> The kat you added by right click isn't actually a kat~! It's a don.
9. Edit your error laden masterpiece
> Kats by right clicking turned into Dons :(
10. Add a new kat by right clicking
11. Use Select Tool and highlight that kat
> Notice how it doesn't add in a Clap or Whistle

osu!lazer version: 2021.515.0

Remember to NOT skip steps 1-3, because that just causes another bug that might already be reported.

**Comment by peppy** [comment]
right clicking isn't intended to be used to place objects. i think you may be after a live mapping mode?

**Comment by ArztVielfrass** [comment]
![image](https://user-images.githubusercontent.com/29100055/118817694-ba841b80-b870-11eb-80f6-3129aa4d387f.png)

Right clicking for kat notes isn't intended? Then this is a UX issue then, I don't want to select the dons I just added to add a clap to them. I was mapping a song on Taiko with the song paused, using LMBs and RMBs to add in notes, I only noticed the error when all the notes I placed were dons instead.
